website/src/components/about/AboutPage.js

```diff
@@ -46,7 +46,7 @@ function AboutPage() {
       `,
     "How often is the data updated?": "The data is fetched from the sources every few hours, but it is updated only when the data has changed.",
     "Can I get access to the data?": "You can explore the current GraphQL API at <a href='/graphql'>Bedav GraphQL API</a>. However, this is just meant for the website. I am currently working on an API that everyone can use to get access to all the data available.",
-    "Can I contribute?": "Yes, you can! Email me at <a href='mailto:shreyas.sreenivasa@gmail.com'>shreyas.sreenivasa@gmail.com</a>",
+    "Can I contribute?": "Yes, you can! Go to the <a href='https://github.com/shreyas44/bedav' target='_blank'>official github repository</a> or email me at <a href='mailto:shreyas.sreenivasa@gmail.com'>shreyas.sreenivasa@gmail.com</a>",
```

Comment:
Could you change it "You can find our GitHub repository [here](https://github.com/shreyas44/bedav) or you can email me at ..."

Reply:
Sure!